src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -72,9 +72,15 @@ private static Window InternalGetCurrentWindow()
 			return _current;
 		}
 
-		internal void RaiseNativeSizeChanged(int screenWidth, int screenHeight)
+		internal void RaiseNativeSizeChanged()
 		{
-			var newBounds = ViewHelper.PhysicalToLogicalPixels(new Rect(0, 0, screenWidth, screenHeight));
+			var display = (ContextHelper.Current as Activity)?.WindowManager?.DefaultDisplay;
+			var fullScreenMetrics = new DisplayMetrics();
+
+			display?.GetRealMetrics(outMetrics: fullScreenMetrics);
```

Comment:
You may want to explain the difference in behavior with Resources.DisplayMetrics is not appropriate here.

Reply:
Just did it